### ISSUE TITLE
[BUG] Modular Armor hud symbol not displaying properly.

### ISSUE THREAD

**Issue by NolanHewitt** [GabrielOlvH/Industrial-Revolution#564]
![ezgif-5-757d8077f8](https://github.com/GabrielOlvH/Industrial-Revolution/assets/52480898/0dbeba69-5f29-403f-8f13-91fd7d6d89ac)

I also tested this using no mods other than the ones required to run Indrev using the most recent mod loader for both fabric and quilt which did not fix the issue. Issues remains on both fabric loader and quilt loader.

**Comment by Pitagoraaa** [comment]
I have the same situation with 1.20.1/Fabric.
I feel uncomfortable with the transparency/lightness of the GUI.

**Comment by kharom1** [comment]
Yup can confirm, also the chat gets transparent, is there a way to disable that feature or something?

**Comment by NolanHewitt** [comment]
Everyday I hope for a fix for this. My only issue with this mod.

**Comment by Flareonn** [comment]
> Everyday I hope for a fix for this. My only issue with this mod.

Hi! I think I managed to remove the hud from the screen!

Try this: find the hud.json file in the game folder on this path: config/indrev/hud.json

And set these values there, as in the screenshot. Well, or any other astronomically huge numbers so that the HUD is rendered outside the field of view of your monitor

![image](https://github.com/user-attachments/assets/1fa135f5-d313-4426-8a9f-215ae001d05f)

**Comment by NolanHewitt** [comment]
> > Everyday I hope for a fix for this. My only issue with this mod.
> 
> Hi! I think I managed to remove the hud from the screen!
> 
> Try this: find the hud.json file in the game folder on this path: config/indrev/hud.json
> 
> And set these values there, as in the screenshot. Well, or any other astronomically huge numbers so that the HUD is rendered outside the field of view of your monitor
> 
> ![image](https://private-user-images.githubusercontent.com/35931685/369603265-1fa135f5-d313-4426-8a9f-215ae001d05f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5MjAzNTUsIm5iZiI6MTcyNjkyMDA1NSwicGF0aCI6Ii8zNTkzMTY4NS8zNjk2MDMyNjUtMWZhMTM1ZjUtZDMxMy00NDI2LThhOWYtMjE1YWUwMDFkMDVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIxVDEyMDA1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ1ZTJjNTZiODJiZDU0M2Q3YjJkMThiNWNiYTdhNzIxYWQ3M2E0YmZiYzIzNTYwYWIwYTA5NzViZjc1YmE3NjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.EyOufcd5EjSYlq097T3Ksa258a222YQHUVhxHcLarm4)

The only issue with that would be that you wouldn’t be able to tell how much shields you have left

**Comment by Flareonn** [comment]
I don't see the need for this, there is Powah! in my modpack which charges armor in all dimensions

**Comment by NolanHewitt** [comment]
> I don't see the need for this, there is Powah! in my modpack which charges armor in all dimensions

No not the energy, the protective shields from the protection module that protects you from damage. In the middle of combat you wouldn’t know how much damage you can take without the hud.
